ansible-ipi-install/README.md

```diff
@@ -321,7 +321,7 @@ provisioner.example.com
 # The following cert_* variables are needed to create the certificates
 #   when creating a disconnected registry. They are not needed to use
 #   an existing disconnected registry.
-# cert_country=MyCountry
+# cert_country=ES #it must be two letters country
```

Comment:
can we default it to US? (don't hate me lol)

Reply:
Yes, it is already where US are, that is ES ;-)

Reply:
Will update it tomorrow